WordPress/src/main/res/layout/backup_download_details_fragment.xml

```diff
@@ -1,70 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/backup_details_scroll_view"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Question (❓): The commit for this file change said `Rename styles to backup download for reuse`, but I saw that there was a bit of deletion happened as part of this commit for this xml file, what was the reason for that? Since the new functionality is not hooked up yet, I suggest that this UI stay as is, until it gets replaced. Otherwise, on the next PR it might look as if it was created from scratch. Wdyt?

Reply:
> Question (❓): 

Ah, the commit first changed the styles and then the next one changed to a recyclerView. Either way, this file is just a recyclerView and the parts that were deleted are truly gone.